Packs/CommonPlaybooks/Playbooks/playbook-User_Investigation_-_SIEM_search_for_failed_logon_-_Generic.yml

```diff
@@ -0,0 +1,706 @@
+id: User Investigation - SIEM search for failed logon - Generic
+version: -1
+name: User Investigation - SIEM search for failed logon - Generic
+description: |-
+  This playbook searches for failed logon on a specific user by querying logs from different sources.
+
+  Supported Integrations:
+  -Splunk
+  -QRadar
+  -Azure Log Analytics
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: ffe1835b-ae14-4728-88fb-2feb4bbfcf2c
+    type: start
+    task:
+      id: ffe1835b-ae14-4728-88fb-2feb4bbfcf2c
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "3"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -440,
+          "y": 150
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: db1d4f2f-0320-4836-8f7b-9bc2af367ca3
+    type: title
+    task:
+      id: db1d4f2f-0320-4836-8f7b-9bc2af367ca3
+      version: -1
+      name: Splunk
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "5"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 230,
+          "y": 550
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: a56e6b38-5c88-4401-83b8-56fa0c6c1fb7
+    type: title
+    task:
+      id: a56e6b38-5c88-4401-83b8-56fa0c6c1fb7
+      version: -1
+      name: QRadar
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "4"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -440,
+          "y": 550
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 4e18dce1-f4b1-470e-8848-9fbacc4045d0
+    type: condition
+    task:
+      id: 4e18dce1-f4b1-470e-8848-9fbacc4045d0
+      version: -1
+      name: Is SIEM enabled?
```

Comment:
You don't have else condition in the playbook. What will happen is none of the SIEM instances enabled?

Reply:
Done.